test-apps/tutorial-servo-can-control/src/TutorialServo.cpp

```diff
@@ -0,0 +1,30 @@
+#include "TutorialServo.h"
+#include "mbed.h"
+
+//I believe we are writing to servoPin, which I set to be PA_1
+CAN can1(PA_0, servoPin);
+float pulseWidthToSend;
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees = 180.0, float minPulsewidthInMs = 1, float maxPulsewidthInMs = 2):m_servoPwmOut(servoPin), m_servoRangeInDegrees(servoRangeInDegrees), m_minPulsewidthInMs(minPulsewidthInMs), m_maxPulsewidthInMs(maxPulsewidthInMs) {
+    m_servoPwmOut.period_ms(20);
+}
+
+// Set servo position (ex. 45 deg)
+void setPositionInDegrees( const float degrees) {
+    m_servoPwmOut.pulsewidth((1+degrees/servoRangeInDegrees) / 1000);
```

Comment:
Keep in mind the servo min / max pulse widths are variable and are stored in `m_minPulseWidthInMs` and `m_maxPulsewidthInMs`. You should be using those values to map from degrees to pulsewidth.

Reply:
Also consider if the value of `degrees` is negative or greater than the servo range